target/linux/ath79/base-files/lib/preinit/03_swap_iface_names_ath79

```diff
@@ -0,0 +1,14 @@
+
+preinit_swap_ethernet_names() {
```

Comment:
i dont understand the purpose of this file. please drop it and use eth1 as lan and eth0 as wan

Reply:
I was maintaining same ethX naming as with ar71xx and stock.  If that doesn't matter, than I can drop it (and/or change ar71xx to match).